api/v01/api_base.rb

```diff
@@ -41,11 +41,12 @@ def self.count_locations(obj)
         end
       end
 
-      def self.count_route_locations(params)
+      # Calculate route for given points considering start and end are not the same
+      def self.count_routes(params)
```

Comment:
`count_route_locations` was better but `count_route_legs` would be even better IMHO because we are not counting the "routes", we are counting the "legs" of the routes. https://en.termwiki.com/EN/leg_(route) So route with A, B, C, D locations would have 3 legs (A-B, B-C, C-D)..

Reply:
Indeed `count_route_legs` is better.